Signal/src/util/UIFont+OWS.m

```diff
@@ -12,19 +12,35 @@
 @implementation UIFont (OWS)
 
 + (UIFont *)ows_thinFontWithSize:(CGFloat)size {
-    return [UIFont systemFontOfSize:size weight:UIFontWeightThin];
+    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(8, 2)) {
+        return [UIFont systemFontOfSize:size weight:UIFontWeightThin];
```

Comment:
Apple changes the system font every so often, so I think we're right to use the system font methods when possible, but we need to fall back to hard-coded font names before 8.2.